src/script/assets/AssetService.ts

```diff
@@ -39,7 +39,8 @@ export interface CompressedImage {
 }
 
 export interface AssetUploadOptions {
-  expectsReadConfirmation: boolean;
+  expectsReadConfirmation?: boolean;
```

Comment:
So far `expectsReadConfirmation` has been mandatory. Why do you want to make it optional? Isn't it good to have it always being `true` or `false`?

Reply:
The default will be falsy, so we can omit it, too. Just wanted to make the API easier :slightly_smiling_face:

Reply:
I would rather have a pure `false` than working with falsy to be explicit.

Reply:
Done!